core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAOSpec.scala

```diff
@@ -218,39 +185,6 @@ class BillingProfileManagerDAOSpec extends AnyFlatSpec with MockitoSugar {
     assertResult(Seq(expectedApp))(apps)
   }
 
-  it should "return all profiles from listBillingProfiles when the profiles exceeds the request batch size" in {
```

Comment:
I don't think we want to lose this test case.

Reply:
Thanks; this was an intermediate diff, the test is back.